deploy/crds/worker-0.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: worker-0-bmc-secret
+type: Opaque
+data:
+  username: YWRtaW4=
+  password: cGFzc3dvcmQ=
+
+---
+apiVersion: metalkube.org/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: worker-0
+spec:
+  online: true
+  bmc:
+    address: libvirt://192.168.122.1:6233/
```

Comment:
I wonder if using `libvirt` schema will make people assume we actually use `libvirt` in ironic (as opposed to using IPMI-to-libvirt).

Reply:
Only developers will ever see it. I think we're OK, at least for now.